core/test/fixtures/fraggle-rock/reports/sample-flow-result.json

```diff
@@ -8516,84 +8516,8 @@
             "title": "Keep request counts low and transfer sizes small",
             "description": "To set budgets for the quantity and size of page resources, add a budget.json file. [Learn more about performance budgets](https://web.dev/use-lighthouse-for-performance-budgets/).",
             "score": null,
-            "scoreDisplayMode": "informative",
-            "displayValue": "18 requests • 65 KiB",
-            "details": {
-              "type": "table",
-              "headings": [
-                {
-                  "key": "label",
-                  "valueType": "text",
-                  "label": "Resource Type"
-                },
-                {
-                  "key": "requestCount",
-                  "valueType": "numeric",
-                  "label": "Requests"
-                },
-                {
-                  "key": "transferSize",
-                  "valueType": "bytes",
-                  "label": "Transfer Size"
-                }
-              ],
-              "items": [
-                {
-                  "resourceType": "total",
-                  "label": "Total",
-                  "requestCount": 18,
-                  "transferSize": 66067
-                },
-                {
-                  "resourceType": "image",
-                  "label": "Image",
-                  "requestCount": 10,
-                  "transferSize": 31673
-                },
-                {
-                  "resourceType": "other",
-                  "label": "Other",
-                  "requestCount": 5,
-                  "transferSize": 31109
-                },
-                {
-                  "resourceType": "script",
-                  "label": "Script",
-                  "requestCount": 3,
-                  "transferSize": 3285
-                },
-                {
-                  "resourceType": "stylesheet",
-                  "label": "Stylesheet",
-                  "requestCount": 0,
-                  "transferSize": 0
-                },
-                {
-                  "resourceType": "media",
-                  "label": "Media",
-                  "requestCount": 0,
-                  "transferSize": 0
-                },
-                {
-                  "resourceType": "font",
-                  "label": "Font",
-                  "requestCount": 0,
-                  "transferSize": 0
-                },
-                {
-                  "resourceType": "document",
-                  "label": "Document",
-                  "requestCount": 0,
-                  "transferSize": 0
-                },
-                {
-                  "resourceType": "third-party",
-                  "label": "Third-party",
-                  "requestCount": 2,
-                  "transferSize": 25328
-                }
-              ]
-            }
+            "scoreDisplayMode": "error",
+            "errorMessage": "Required LinkElements gatherer did not run."
```

Comment:
what am i missing @adamraine ?

Reply:
You need to add `LinkElements` to the checked in flow artifacts. It wasn't run in timespan mode before because of filtering I guess.

Reply:
Is there a command? something like `update:sample-artifacts` but for flow

Reply:
`yarn update:flow-sample-json -a LinkElements`